packages/syntax-floating-components/src/Popover/Popover.tsx

```diff
@@ -1,263 +1,164 @@
-  /**
-   * Location of the Popover content relative to anchor element
-   *
-   * @defaultValue "right"
-   */
-  placement?: Side;
-  /**
-   * Placement strategy, either "fixed" or "absolute"
-   *
-   * @defaultValue "absolute"
+   * Location of the popover content relative to anchor element
+   * @defaultValue "top-start"
    */
-  strategy?: Strategy;
-};
-
-export function usePopover({
-  delay = 0,
-  initialOpen = false,
-  open: controlledOpen,
-  placement = "right",
-  strategy = "absolute",
-  onOpen = undefined,
-  onClose = undefined,
-}: PopoverOptions): UseFloatingReturn & {
-  getReferenceProps: (
-    userProps?: React.HTMLProps<Element> | undefined,
-  ) => Record<string, unknown>;
-  getFloatingProps: (
-    userProps?: React.HTMLProps<HTMLElement> | undefined,
-  ) => Record<string | number | symbol, unknown>;
-  getItemProps: (
-    userProps?: React.HTMLProps<HTMLElement> | undefined,
-  ) => Record<string | number | symbol, unknown>;
-  arrowRef: React.MutableRefObject<null>;
-  open: boolean;
-  setOpen: (open: boolean) => void;
-} {
-  const [uncontrolledOpen, setUncontrolledOpen] = React.useState(initialOpen);
-
-  const arrowRef = React.useRef(null);
-
-  const open = controlledOpen ?? uncontrolledOpen;
-  let setOpen: (open: boolean) => void;
-  if (onOpen && controlledOpen == true) {
-    setOpen = onOpen;
-  } else if (onClose && controlledOpen == false) {
-    setOpen = onClose;
-  } else {
-    setOpen = setUncontrolledOpen;
-  }
-
-  const data = useFloating({
-    placement,
-    strategy,
-    open,
-    onOpenChange: setOpen,
-    whileElementsMounted: autoUpdate,
-    middleware: [
-      offset(8),
-      flip({
-        fallbackAxisSideDirection: "start",
-      }),
-      shift({ padding: 4 }),
-      arrow({ element: arrowRef }),
-      size({
-        apply({ availableHeight, elements }) {
-          Object.assign(elements.floating.style, {
-            maxHeight: `${availableHeight}px`,
-          });
-        },
-      }),
-    ],
-  });
-
-  const context = data.context;
-
-  const hover = useHover(context, {
-    move: false,
-    enabled: controlledOpen == null,
-    delay,
-  });
-  const focus = useFocus(context, {
-    enabled: controlledOpen == null,
-  });
-  const dismiss = useDismiss(context);
-  const role = useRole(context, { role: "dialog" });
-
-  const interactions = useInteractions([hover, focus, dismiss, role]);
-
-  return React.useMemo(
-    () => ({
-      open,
-      setOpen,
-      ...interactions,
-      ...data,
-      arrowRef,
-    }),
-    [open, setOpen, interactions, data],
-  );
-}
-
-const PopoverContext = React.createContext<ReturnType<
-  typeof usePopover
-> | null>(null);
-
-const usePopoverContext = () => {
-  const context = React.useContext(PopoverContext);
-
-  if (context == null) {
-    throw new Error("Popover components must be wrapped in <Popover />");
-  }
-
-  return context;
+  placement?: PopoverStoreProps["placement"];
```

Comment:
@christianvuerings I think this will still be something to comment on in the new react-aria based pr.  We'll probably want to consolidate the Placement prop between Tooltip + Popover.  Keep an eye out for the discrepency in that PR and lmk if you have a lightbulb go off for a clean way to consolidate them 🙏